php-practice/section5.php

```diff
@@ -142,7 +148,11 @@ function printLines(array $lines) {
  *
  * Explain in your own words how the above function works.
  *
- * ANSWER:
+ * ANSWER: This function (called printLines) takes two parameters: the array and the variable $lines. We then use the foreach construct to loop through our array
+ * and print each value on its on line. The foreach loop takes our variable $lines and for each value in the array will place it in the $line each time. We also
+ * need to tell the foreach construct what we wnat it to do once it has placed the value in the variable $line. In this case, we want it to print out the value on its own line.
+ * Now that we have defined our function and construct, we pass the array values using "echo printLines" followed by the array values. We can pass multiple arrays which will all
+ * be printed on different lines.
```

Comment:
1. I was trying to be fancy and I noticed that "foreach" is called a construct in the PHP manual. I tried finding the reason why they called the "foreach loop" a construct and couldn't find it.
2. We can "echo" multiple arrays is what I meant. However, to answer your questions, we can "call" the functions multiple times using "echo printLines", we can call it 1 time and pass it multiple arrays (I am just not sure what the syntax for that would look like).